packit/schema.py

```diff
@@ -430,10 +430,9 @@ def ordered_preprocess(self, data, **_):
                 )
 
         for key in ("targets", "dist_git_branches"):
-            if data is dict:
```

Comment:
Ya… this works in C# or Rust for matching… It's contradiction in Python…

Reply:
Actually, I think this check can make sense, loading a YAML can also result in a list or string, which is not something we should accept, but things shouldn't fail here either (they will fail at schema validation).

So I fixed this check instead of removing it. Still looks good? :)